src/dodal/devices/oav/oav_detector.py

```diff
@@ -70,6 +83,102 @@ def set(self, level_to_set: str) -> StatusBase:
         return return_status
 
 
+class OAVConfigParams:
+    """
+    The OAV parameters which may update depending on settings such as the zoom level.
+    """
+
+    zoom: float
+
+    def __init__(
+        self,
+        zoom_params_file=ZOOM_PARAMS_FILE,
+        display_config=DISPLAY_CONFIG,
```

Comment:
Should: Now that we're creating one of these in each beamline file we should remove the defaults here